pkg/api/testapi/testapi.go

```diff
@@ -79,6 +79,10 @@ func init() {
 	Extensions = Groups[extensions.GroupName]
 }
 
+func (g TestGroup) ContentConfig() (string, *unversioned.GroupVersion, runtime.Codec) {
+	return "application/json", g.GroupVersion(), g.Codec()
+}
+
```

Comment:
Hmm.. To me, this use case suggest that perhaps content type should be part of the codec. Thoughts?